auth-web/tests/unit/views/IncorpOrRegisterView.spec.ts

```diff
@@ -47,7 +47,7 @@ describe('IncorpOrRegisterView.vue', () => {
 
   it('renders the components properly', () => {
     expect(wrapper.find(IncorpOrRegisterView).exists()).toBe(true)
-    expect(wrapper.find(NumberedCompanyTooltip).exists()).toBe(true)
+    expect(wrapper.find(NumberedCompanyTooltip).exists()).toBe(false) // Currently Pending Verbiage
```

Comment:
How are you keeping track of future work here?

Reply:
Yessir, we have ticket #3853 to tackle all these little things still outstanding before we launch. Will revisit them at that point! The tooltip in question is one of those things.
 I fully expect to use it, tho they are still working on the approved copy of the verbiage afaik. I just removed the tool tip for the time being, in the event that it isn't decided for MVP.

Reply:
Awesome. Well done :)

Reply:
Resolved.